go/dbproxy/main.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"almond-cloud/sql"
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+)
+
+func main() {
+	dsn := "newuser:password@tcp(127.0.0.1:3306)/dbname?charset=utf8mb4&parseTime=True&loc=Local"
+	sql.InitMySQL(dsn)
```

Comment:
I assume we'll make the Go code read the same config files as the node backend, correct?

Reply:
Yes, will do.